ci/ci/constants.py

```diff
@@ -3,7 +3,7 @@
 
 GITHUB_CLONE_URL = 'https://github.com/'
 
-with open(os.environ['HAIL_TOKEN_FILE']) as f:
+with open(os.environ.get('HAIL_TOKEN_FILE') or os.path.expanduser('~/.hail/token')) as f:
```

Comment:
this now appears in aioclient and here. Let's pull it into gear.

```python
def find_token(token_file=None):
    token_file = (token_file or
                  os.environ.get('HAIL_TOKEN_FILE') or
                  os.path.expanduser('~/.hail/token'))
    if not os.path.exists(token_file):
       raise ValueError(
           f'Cannot create a client without a token. No file was '
           f'found at {token_file}')
    with open(token_file) as f:
        return f.read()

def find_userdata(token_file=None):
    return hj.JWTClient.unsafe_decode(find_token(token_file))
```

maybe add them to JWTClient.